src/main/java/uk/gov/ons/census/fwmt/jobservice/service/converter/ccs/CcsInterviewCreateConverter.java

```diff
@@ -0,0 +1,75 @@
+package uk.gov.ons.census.fwmt.jobservice.service.converter.ccs;
+
+import org.springframework.beans.factory.annotation.Value;
+import uk.gov.ons.census.fwmt.common.data.tm.Address;
+import uk.gov.ons.census.fwmt.common.data.tm.CaseRequest;
+import uk.gov.ons.census.fwmt.common.data.tm.CaseType;
+import uk.gov.ons.census.fwmt.common.data.tm.CcsCaseExtension;
+import uk.gov.ons.census.fwmt.common.data.tm.Contact;
+import uk.gov.ons.census.fwmt.common.data.tm.Geography;
+import uk.gov.ons.census.fwmt.common.data.tm.SurveyType;
+import uk.gov.ons.census.fwmt.common.rm.dto.FwmtActionInstruction;
+import uk.gov.ons.census.fwmt.jobservice.data.GatewayCache;
+import uk.gov.ons.census.fwmt.jobservice.service.converter.common.CommonCreateConverter;
+
+import java.util.List;
+import java.util.Objects;
+
+public class CcsInterviewCreateConverter  {
+
+  public static String eqUrl;
+
+  private CcsInterviewCreateConverter(@Value("${eq.url}") String eqUrl) {
+    CcsInterviewCreateConverter.eqUrl = eqUrl;
+  }
+
+  public static CaseRequest.CaseRequestBuilder convertCcs(
+      FwmtActionInstruction ffu, GatewayCache cache, CaseRequest.CaseRequestBuilder builder) {
+    CaseRequest.CaseRequestBuilder commonBuilder = CommonCreateConverter.convertCommon(ffu, cache, builder);
+
+    commonBuilder.type(CaseType.valueOf(ffu.getAddressType()));
+    commonBuilder.surveyType(SurveyType.CCS_INT);
+    commonBuilder.category(ffu.getAddressType().equals("HH") ? "HH" : "CE");
```

Comment:
could AddressType ever be null?
If so `"HH".equals(ffu.getAddressType())` would be safer

Reply:
okay incase